test/spec/asyncapi.yml

```diff
@@ -39,16 +39,16 @@ channels:
       summary: Inform about environmental lighting conditions of a particular streetlight.
       operationId: receiveLightMeasurement
       tags:
-        - name: oparation-tag1
+        - name: operation-tag1
           externalDocs: 
             description: External docs description 1
             url: https://www.asyncapi.com/
-        - name: oparation-tag2
+        - name: operation-tag2
           description: Description 2
           externalDocs:
             url: "https://www.asyncapi.com/"
-        - name: oparation-tag3
-        - name: oparation-tag4
+        - name: operation-tag3
+        - name: operation-tag4
           description: Description 4
         - name: message-tag5
```

Comment:
can you also remove this, it also should not be here, an old mistake

Reply:
The `name`s you mean? 👀

Reply:
entire tag

Reply:
There it is. Sorry for the delay.

Reply:
no worries mate, thanks for this additional change but I think I was misunderstood, sorry, by `entire tag` I meant the tag that I commented, so just
```
        - name: message-tag5
          externalDocs:  
            url: "https://www.asyncapi.com/"
```

the rest of tags should stay, sorry again

Reply:
Cool, no problemo. Just fixed and squashed the commits, thanks @derberg.